content/v2.0/reference/key-concepts.md

```diff
@@ -0,0 +1,184 @@
+
+| _time                 | _measurement | _field | _value |<span class="tooltip" data-tooltip-text="Tag key">bees</span>|<span class="tooltip" data-tooltip-text="Tag key">ants</span>|
+|:----------------------|--------------|--------|--------|-------|-------|
+| 2019-08-18T00:00:00Z | census |scientist |anderson   |   23 | |
+| <span class="tooltip" data-tooltip-text="Timestamp">2015-08-18T00:00:00Z</span> | <span class="tooltip" data-tooltip-text="Measurement name">census</span> |<span class="tooltip" data-tooltip-text="Field key">scientist</span> | <span class="tooltip" data-tooltip-text="Field value">mullen</span> | <span class="tooltip" data-tooltip-text="Tag value">30</span>  |
+| 2019-08-18T00:06:00Z | census |scientist| anderson| 28 |    |
+|                      |        |         |         |    |    |
+| 2019-08-18T00:00:00Z | census |location | 1       | 23 |    |
+| 2019-08-18T00:00:00Z | census |location | 1       | 30 |    |
+| 2019-08-18T00:06:00Z | census |location | 2       | 28 |    |
+|                      |        |         |         |    |    |
+| 2019-08-18T00:06:00Z | census |location | 2       |    | 3  |
+
+Now that `bees` and `ants` are tags, InfluxDB doesn't have to scan all `_field` and `_value` columns. This makes your queries faster.
+
+#### Series
+
+Now that you're familiar with measurements, field sets, and tag sets, it's time to discuss **series keys** and **series**. A series key is the collection of data that shares a measurement, tag set, and field key. For example, the [sample data](#sample-data) includes four unique series:
+
+| _measurement  | tag set                         | _field |
+|:-------------|:---------------------------------|:-------|
+| census       |<span class="tooltip" data-tooltip-text="Tag 1">location = 1</span>,scientist = anderson|<span class="tooltip" data-tooltip-text="Field key">bees</span>|
+| census       |location = 2,<span class="tooltip" data-tooltip-text="Tag 2">scientist = anderson</span>  |bees |
+| census       |location = 1,scientist = mullen   |bees|
+| census       |location = 2,scientist = mullen   |ants|
+
+A **series** is a group of field values for a unique series key. In a series, field values (`_values`) are ordered by timestamp (`_time`) in ascending order.
+
+| _time                     | _values     |
+|---------------------------|-------------|
+| `2019-08-18T00:00:00Z`    | `23`        |
+| `2019-08-18T00:00:00Z`    | `30`        |
+| `2019-08-18T00:06:00Z`    | `28`        |
+| `2019-08-18T00:06:00Z`    | `3`         |
+
+Understanding the concept of a series is essential when designing your [schema](v2.0/reference/glossary/#schema) and when working with your data in InfluxDB.
```

Comment:
Understanding the concept of a series is essential when designing your [schema](v2.0/reference/glossary/#schema) and ~when~ working with your data in InfluxDB.